tests/test_db_runtime_del_coverage.py

```
@@ -0,0 +1,71 @@
+
+    def test_del_suppresses_close_exception(self, tmp_path):
+        db_path = str(tmp_path / "test.db")
+        mgr = DatabaseManager(db_path)
+        try:
+            with patch.object(mgr, "close", side_effect=RuntimeError("boom")):
+                mgr.__del__()
+        finally:
+            try:
+                mgr.close()
+            except Exception:
+                pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)